include/camp/resource/sycl.hpp

```diff
@@ -197,7 +197,27 @@ namespace resources
         }
       }
 
-      sycl::queue *get_queue() { return qu; }
+      sycl::queue *get_queue() const { return qu; }
```

Comment:
Thanks for adding const in a number of appropriate places, but this one needs a bit of repair.  It needs to have both variants and return a const pointer when const.
```suggestion
      sycl::queue *get_queue() { return qu; }
      sycl::queue const *get_queue() const { return qu; }
```

Reply:
Sounds good!